velox/docs/functions/spark/binary.rst

```diff
@@ -5,18 +5,30 @@ Binary Functions
 .. spark:function:: hash(x, ...) -> integer
 
     Computes the hash of one or more input values using seed value of 42.
+    Supports types including: BOOLEAN, TINYINT, SMALLINT, INTEGER, BIGINT,
```

Comment:
Supported types are: ...

Remove "Not supports types..."

Add something like "Different arguments may have different types."

Reply:
Updated.